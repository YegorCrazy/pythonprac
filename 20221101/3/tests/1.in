a = Grange(1, 3, 30)
print(a)
print(len(a))
if not a:
    print("Empty")
print(*[a[i] for i in range(6)], sep=' ')
print(*list(a), sep=' ')
print(*list(a[2:10]), sep=' ')
print(*list(a[2:20:2]), sep=' ')
