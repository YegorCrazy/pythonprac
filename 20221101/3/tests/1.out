grange(1, 3, 30)
4
1 3 9 27 81 243
1 3 9 27
2 6
2 18
